chronicles/log_output.nim

```diff
@@ -56,16 +56,14 @@ type
   AnyFileOutput = FileOutput|StdOutOutput|StdErrOutput
   AnyOutput = AnyFileOutput|SysLogOutput|BufferedOutput|PassThroughOutput
 
-  TextLineRecord*[Output;
-                  timestamps: static[TimestampsScheme],
-                  colors: static[ColorScheme]] = object
-    output*: Output
-    level: LogLevel
-
-  TextBlockRecord*[Output;
-                   timestamps: static[TimestampsScheme],
-                   colors: static[ColorScheme]] = object
+  LogRecord*[Output;
+           logformat: static[LogFormat],
```

Comment:
I'm sorry my explanation wasn't clear enough. The names of the formats such as `Json`, `TextBlock` and `TextLine` are Nim type names. You don't need to (and you shouldn't) pass them as `static` parameters.

The correct definition is 

```nim
type
  LogRecord[Output; Format; timestamps: static[TimestampsScheme], ...] = object
     writer: WriterType(Format)
```